test/Elastica/Query/DistanceFeatureTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Elastica\Test\Query;
+
+use Elastica\Query\DistanceFeature;
+use Elastica\Test\Base as BaseTest;
+
+class DistanceFeatureTest extends BaseTest
+{
+    /**
+     * @group unit
+     */
+    public function testToArrayDate()
+    {
+        $query = new DistanceFeature('field_date', 'now', '7d');
+
+        $expectedArray = [
+            'distance_feature' => [
+                'field' => 'field_date',
+                'origin' => 'now',
+                'pivot' => '7d',
+            ],
+        ];
+
+        $this->assertEquals($expectedArray, $query->toArray());
+    }
+
+    /**
+     * @group unit
+     */
+    public function testToArrayGeoPoint()
```

Comment:
Any chance to also add a functional test? This helps in case ES breaks things, we will know.